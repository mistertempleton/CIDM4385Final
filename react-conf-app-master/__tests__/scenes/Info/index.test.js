// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Info from '../../../app/scenes/Info';

describe('Info', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Info navigator={{}} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
