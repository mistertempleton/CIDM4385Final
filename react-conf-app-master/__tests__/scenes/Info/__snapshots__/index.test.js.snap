exports[`Info renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }>
  <View>
    <View
      style={
        Object {
          "height": 64,
        }
      } />
    <View
      style={
        Array [
          Object {
            "alignItems": "stretch",
            "borderBottomColor": "#CCC",
            "borderBottomWidth": 0.5,
            "flexDirection": "row",
            "height": 64,
            "justifyContent": "space-between",
            "overflow": "hidden",
            "paddingTop": 20,
            "position": "absolute",
            "width": 750,
          },
          Object {
            "backgroundColor": "white",
          },
        ]
      }>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 3,
            "flexDirection": "row",
            "paddingHorizontal": 17,
          }
        } />
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 4,
            "justifyContent": "center",
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 17,
                "fontWeight": "500",
              },
              Object {
                "color": "#383838",
              },
            ]
          }>
          About
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "flex": 3,
            "flexDirection": "row",
            "justifyContent": "flex-end",
            "opacity": 1,
            "paddingHorizontal": 17,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#00A8D8",
                "fontSize": 17,
              },
              Object {
                "opacity": 1,
              },
            ]
          }>
          Directions
        </Text>
      </View>
    </View>
  </View>
  <RCTScrollView>
    <View>
      <MapView
        initialRegion={
          Object {
            "latitude": 37.391084,
            "latitudeDelta": 0.01,
            "longitude": -121.9746,
            "longitudeDelta": 0.01,
          }
        }
        style={
          Object {
            "flex": 1,
            "height": 200,
            "maxHeight": 200,
          }
        }>
        <Marker
          coordinate={
            Object {
              "latitude": 37.391084,
              "latitudeDelta": 0.01,
              "longitude": -121.9746,
              "longitudeDelta": 0.01,
            }
          }
          description="2700 Mission College Blvd, Santa Clara, CA 95054"
          onCalloutPress={[Function]}
          title="Santa Clara Marriott" />
      </MapView>
      <View
        style={
          Object {
            "flex": 1,
          }
        }>
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "borderBottomColor": "#CCC",
              "borderBottomWidth": 0.5,
              "borderTopColor": "#B3B3B3",
              "borderTopWidth": 0.5,
              "paddingHorizontal": 17,
            }
          }>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 17,
                "fontWeight": "300",
                "lineHeight": 24,
                "paddingTop": 32,
                "textAlign": "center",
              }
            }>
            The conference will be taking place on March 13th and 14th, with talks from 10am to 6pm each day. Plan to hang out with us each evening for plenty of socializing over food and drink.
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 17,
                "fontWeight": "300",
                "lineHeight": 24,
                "paddingTop": 32,
                "textAlign": "center",
              }
            }>
            Proceeds from all ticket sales are being donated to Code2040.
          </Text>
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            hitSlop={undefined}
            isTVSelectable={true}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "color": "#00A8D8",
                  "fontSize": 17,
                  "fontWeight": "500",
                  "padding": 24,
                }
              }>
              Code of Conduct
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "paddingHorizontal": 17,
              "paddingVertical": 32,
            }
          }>
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            hitSlop={undefined}
            isTVSelectable={true}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "opacity": 1,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}>
            <Image
              source={1}
              style={
                Object {
                  "height": 44,
                  "width": 44,
                }
              } />
          </View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "color": "#666",
                "fontSize": 14,
                "fontWeight": "300",
                "lineHeight": 20,
                "marginTop": 17,
                "textAlign": "center",
              }
            }>
            Made with love in Sydney, Australia, and open sourced by Thinkmill
          </Text>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
